tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -281,7 +281,7 @@
     <value>WER Reports</value>
     <comment>The contents of a WER crash dumps.</comment>
   </data>
-  <data name="WERTimeGeneratedColumn.Header" xml:space="preserve">
+  <data name="TimeGeneratedColumn.Header" xml:space="preserve">
     <value>Time generated</value>
     <comment>The time at which the WER report was generated.</comment>

```

Comment:
Good catch, I will update the comment. The string will be same in both pages, so we should be able to share these common strings.

Reply:
Nothing wrong in it, since it's the same string, I thought we could use the same one. I don't have a strong opinion on it, I will create multiple strings.